DataFormats/TotemReco/interface/TotemT2Digi.h

```diff
@@ -0,0 +1,37 @@
+/****************************************************************************
+ *
+ * This is a part of TOTEM offline software.
+ * Author:
+ *   Laurent Forthomme
+ *
+ ****************************************************************************/
+
+#ifndef DataFormats_TotemReco_TotemT2Digi_h
+#define DataFormats_TotemReco_TotemT2Digi_h
+
+class TotemT2Digi {
+public:
+  explicit TotemT2Digi() = default;
```

Comment:
Can you explain, why is this constructor explicit?

Reply:
No particular reason, probably a leftover from a previous constructor with a single VFAT frame argument. Can be removed in a later push.